internal/module/openapi.go

```diff
@@ -66,32 +67,40 @@ func helmFormatModuleImages(m *Module, rawValues map[string]any) (chartutil.Valu
 	return top, nil
 }
 
-func GetModulesImagesDigests(modulePath string) (modulesDigests map[string]any, err error) {
+func GetModulesImagesDigests(moduleName, modulePath string) (modulesDigests map[string]any, err error) {
 	var (
 		search bool
 	)
 
-	if fi, errs := os.Stat(filepath.Join(filepath.Dir(modulePath), imageDigestfile)); errs != nil || fi.Size() == 0 {
-		search = true
-	}
-
-	if search {
-		return DefaultImagesDigests, nil
-	}
-
 	modulesDigests, err = getModulesImagesDigestsFromLocalPath(modulePath)
 	if err != nil {
 		return nil, err
 	}
 
-	return modulesDigests, nil
+	if modulesDigests == nil {
+		if fi, errs := os.Stat(filepath.Join(filepath.Dir(modulePath), imageDigestfile)); errs != nil || fi.Size() == 0 {
```

Comment:
And why check twice for the same error?
We checked the existence of the file in the getModulesImagesDigestsFromLocalPath function.
And if there was no error and modulesDigests is nil, then the file simply did not exist.

Reply:
Simplified in commit [4ac28e1](https://github.com/deckhouse/dmt/pull/25/commits/4ac28e101efe6ac4bcfd622c832e73a7cd5c25a0). 
We need to check if moduleDigest is not nil for case when image digests file is not exists and we can fallback to default one